client/components/SettingsMenu.tsx

```diff
@@ -0,0 +1,58 @@
+'use client';
+import { useTheme } from 'next-themes';
+import { Button } from './ui/button';
+import { User, Sun, Moon, SunMoon } from 'lucide-react';
+import Link from 'next/link';
+import {
+    DropdownMenu,
+    DropdownMenuContent,
+    DropdownMenuItem,
+    DropdownMenuPortal,
+    DropdownMenuSeparator,
+    DropdownMenuSub,
+    DropdownMenuSubContent,
+    DropdownMenuSubTrigger,
+    DropdownMenuTrigger,
+} from './ui/dropdown-menu';
+
+const SettingsMenu = () => {
+    const { setTheme } = useTheme();
+
+    return (
+        <DropdownMenu>
+            <DropdownMenuTrigger asChild>
+                <Button variant="outline">
+                    <User />
+                </Button>
+            </DropdownMenuTrigger>
+            <DropdownMenuPortal>
+                <DropdownMenuContent>
+                    <DropdownMenuSub>
+                        <DropdownMenuSubTrigger>
+                            <SunMoon />
+                            Theme
+                        </DropdownMenuSubTrigger>
+                        <DropdownMenuPortal>
+                            <DropdownMenuSubContent>
+                                <DropdownMenuItem onClick={() => setTheme('light')}>
+                                    <Sun className="pr-0.5" />
+                                    Light
+                                </DropdownMenuItem>
+                                <DropdownMenuItem onClick={() => setTheme('dark')}>
+                                    <Moon className="pr-0.5" />
+                                    Dark
+                                </DropdownMenuItem>
+                            </DropdownMenuSubContent>
+                        </DropdownMenuPortal>
+                    </DropdownMenuSub>
+                    <DropdownMenuSeparator />
+                    <Button className="w-full" variant="destructive">
+                        <Link href="/">Logout</Link>
```

Comment:
This should be "Log Out"

Reply:
I'm also just now realising that the login page says "Login", but should say "Log In".  Can you go ahead and change that in this PR?